examples/chart/teleport-kube-agent/templates/deployment.yaml

```diff
@@ -111,7 +111,7 @@ spec:
           name: "auth-token"
           readOnly: true
         - mountPath: /var/lib/teleport
-          name: "data"
+          name: {{ default "data" .Values.existingDataVolume }}
```

Comment:
Verified that no modern Linux system uses `""` for directory names (relative, system, or otherwise): https://unix.stackexchange.com/questions/160539/when-does-the-empty-string-denote-the-current-directory 

Defaults to "data" if the `existingDataVolume` value doesn't exist. 👍🏼